content/doc/developer/plugin-development/pipeline-integration.adoc

```diff
@@ -17,7 +17,7 @@ there are a number of special considerations.
 
 == Extension points accessible via metastep
 
-Several common types of plugin features (`@Extension` s) can be invoked from a Pipeline script without any special
+Several common types of plugin features (++@Extension++s) can be invoked from a Pipeline script without any special
```

Comment:
able to remove unwanted space

Reply:
@jglick 
Do you still want `@Extension` to be monospaced?
If so, you'll want two backticks as noted in http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#formatted-text .

Reply:
@bitwiseman the backticks in asciidoc doesn't work so well with possessive text. FWIW

Reply:
@jglick @rtyler 
Sorry, the quick reference doesn't cover "+" or "++".  
http://www.methods.co.nz/asciidoc/userguide.html  Gives accurate information about how those two differ from backtick and double backtick. 😢

Reply:
So what is the upshot from https://github.com/asciidoctor/asciidoctor.org/pull/742?

Reply:
@jglick @rtyler 
(Repeating for public consumption)
https://issues.jenkins-ci.org/browse/WEBSITE-428
So, double backtick does't work right now on jenkins.io, but it works everywhere else. 

We should take the step turn off compat mode on jenkins.io and do the search and replace to come up to current.